src/Route/RouteFilter.php

```diff
@@ -102,7 +102,7 @@ public function parseFilterString($filter_string)
 
         $cache_key = $this->generateFilterStringCacheKey($filter_string);
 
-        $this->rules = $this->cache->get($cache_key, []);
+        $this->rules = (array) $this->cache->get($cache_key, []);
```

Comment:
@ssfinney I am not sure that I follow casting it to an array?  If the key is not in the cache, then it returns the default, which we are setting to an empty array.